docs/expansionPort.md

```diff
@@ -0,0 +1,115 @@
+
+<img src="images/amdc-isoSPI.svg" />
+
+### 1. DB15 Connector
+
+A total of two DB15 D-sub high-density connectors are used for both the isoSPI and differential I/O (D_I / D_O) interface. Each connector has pins for 2 isoSPI communication interfaces, 2 differential I/O interfaces, 1 pin for ground signals, and 1 pin for 5V supply from the AMDC. More details on the DB15 connector can be found in [here](https://content.norcomp.net/rohspdfs/Connectors/17Y/178/513/178-H15-513R497.pdf). The location of these connectors in the AMDC is shown below:
+
+<img src="images/amdc-isoSPI-input-highlighted.svg" />
+
+The pin mappings for each DB15 connector is shown below:
+
+| Pin number | Signal name |
+|------------|--------|
+| 1 | 5V |
+| 2 | D1_I_P |
+| 3 | D1_I_N |
+| 4 | D2_I_P |
+| 5 | D2_I_N |
+| 6 | NC |
+| 7 | isoSPI1_P |
+| 8 | isoSPI1_N |
+| 9 | isoSPI2_P |
+| 10 | isoSPI2_N |
+| 11 | GND |
+| 12 | D1_O_P |
+| 13 | D1_O_N |
+| 14 | D2_O_P |
+| 15 | D2_O_N |
+
+The isoSPI signals that connect to the PicoZed pins and Zynq-7000 FPGA module can be found at the isoSPI section of the [pin mapping document](RevD-PinMapping.md#encoder).
+
+### 2. IsoSPI communication interface
+
+The isoSPI communication interface is implemented using the [LTC6820](https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6820.pdf) IC. This IC provides a bi-directional interface between standard SPI signals and differential pulses. It also translates the 1.8V SPI signals from the PicoZed to 5V. For termination, a 120 Ω resistor is added. The maximum and minimum operating conditions are provided in the following table:
+
+
+| Parameter                             |    Conditions     |   MIN   |  MAX  |
+|---------------------------------------|-------------------|---------|-------|
+| Supply voltage VDD (Differential side)|                   | 2.7 V   | 5.5 V |
+| IO supply voltage VDDS (SPI side)     |                   | 1.7 V   | 5.5 V |
+| High-level input voltage              |VDDS = 2.7V to 5V  | 0.7 VDDS| VDDS  |
+| High-level input voltage              |VDDS = 1.7V to 2.7V| 0.8 VDDS| VDDS  |
+| Low-level input voltage               |VDDS = 2.7V to 5V  | 0       | 0.3 VDDS  |
+| Low-level input voltage               |VDDS = 1.7V to 2.7V| 0       | 0.2 VDDS  |
+
+This IC can operate at a maximum SPI communication speed of 1 Mbps. Bias resistors (RB1 and RB2) are used to adjust the drive currents to the differential lines, in this design they set the drive currents to 10 mA. These resistors are on the schematic, see the following figure.
+
+<img src="images/Schematic_RB.PNG" width="200"/>
+
+The maximum supply current consumed by the IC including to drive currents for differential lines is 15.8 mA, which corresponds to 79 mW for a 5 V supply. More information regarding the operating conditions, bias resistors, maximum throughput rate, can be found in the [datasheet](https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6820.pdf). 

```

Comment:
> The maximum supply current consumed by the IC including to drive currents for differential lines is 15.8 mA, which corresponds to 79 mW for a 5 V supply.

It's okay to have this sentence -- but I am not sure how useful or relevant this information is.

Reply:
This will give an idea of how much current these ICs are consuming and compare that to the rating of the supply. This information is present for all ICs in the document. If you think this is not relevant I can remove these sentences.